### PR TITLE
PR : /api/auth/singup controller 구현

### PR DESCRIPTION
**Pull request by unchaptered**

### CODE REVIEW

**Review by axisotherwise — COMMENTED** [review]
here

File: branch.sample.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+console.log("hello");
```
Comment (axisotherwise) [inline branch.sample.ts:1]: here

**Review by axisotherwise — APPROVED** [review]
here

**Comment by unchaptered** [comment]
참고 : [Bug : multipart/form-data 커스텀 파서 만들어보기](https://github.com/cupicks/cupicks-be/issues/4)

**Pull request merged**